app/_data/docs_nav_gateway_2.5.x.yml

```
@@ -204,49 +173,85 @@
         - text: Securing the Admin API
           url: /admin-api/secure-admin-api
     - text: Plugin Development Kit
-      url: /pdk
+      url: /pdk-reference
```

Missed one, this one as well
```suggestion
      url: /pdk/pdk-reference
```